src/constants.ts

```diff
@@ -6,7 +6,8 @@ export enum Inputs {
 }
 
 export enum Outputs {
-    CacheHit = "cache-hit"
+    CacheHit = "cache-hit",
+    CacheKey = "cache-key"
```

Comment:
Would it be better to name it `cache-matched-key` to keep it aligned with the corresponding state being set. 
Not sure if it is allowed to have state and output with same id

Reply:
I will use whatever people request. But, I'd like to avoid updates unless people agree on direction.